content/blog/scale-kubernetes-cluster-using-hpe-greenlake-terraform-provider.md

```diff
@@ -0,0 +1,246 @@
+
+Below is the reference Terraform configuration for creating the cluster with additional nodes.
+
+```hcl
+terraform {
+  required_providers {
+    hpegl = {
+      source  = "hpe/hpegl"
+      version = ">= 0.2.2"
+    }
+  }
+}
+ 
+provider hpegl {
+  caas {
+  }
+}
+ 
+variable "HPEGL_SPACE" {
+  type = string
+}
+ 
+data "hpegl_caas_site" "blr" {
+  name = "BLR"
+  space_id = var.HPEGL_SPACE
+ }
+ 
+data "hpegl_caas_cluster_blueprint" "bp" {
+  name = "demo"
+  site_id = data.hpegl_caas_site.blr.id
+}
+ 
+data "hpegl_caas_machine_blueprint" "standard_worker" {
+  name = "standard-worker"
+  site_id = "data.hpegl_caas_site.blr.id"
+}
+
+resource hpegl_caas_cluster test {
+  name         = "tf-test"
+  blueprint_id = data.hpegl_caas_cluster_blueprint.bp.id
+  site_id      = data.hpegl_caas_site.blr.id
+  space_id     = var.HPEGL_SPACE
+
+  worker_nodes {
+    name = "test-node-pool"
+    machine_blueprint_id = data.hpegl_caas_machine_blueprint.standard_worker.id
+    count = "1"
+  }
+}
+```
+
+
+
+*Note: Machine blueprints are used to define the infrastructure details for the worker nodes used in a cluster. A machine blueprint includes the following:*
+
+* *Machine provider*
+* *Operating system image and version*
+* *Number of vCPU cores and amount of memory in the node*
+
+# Terraform ready to plan
+
+Terraform plan is a dry run that lets you preview the changes that Terraform plans to make to your infrastructure based on the data you provide in your Terraform file. To see this, run: **terraform plan**
+
+```shellsession
+$ terraform plan
+
+hpegl_caas_cluster.test: Refreshing state... [id=a32fabb9-7c19-42d1-9a38-ebf122810c0a]
+
+Terraform used the selected providers to generate the following execution plan. Resource actions are indicated with the following symbols:
+  ~ update in-place
+
+Terraform will perform the following actions:
+
+  # hpegl_caas_cluster.test will be updated in-place
+  ~ resource "hpegl_caas_cluster" "test" {
+        id                    = "a32fabb9-7c19-42d1-9a38-ebf122810c0a"
+        name                  = "tf-test"
+        # (17 unchanged attributes hidden)
+
+      + worker_nodes {
+          + count                = 1
+          + machine_blueprint_id = "0ac21c99-2fdb-491d-a590-a5016690b80b"
+          + name                 = "test-node-pool"
+        }
+    }
+
+Plan: 0 to add, 1 to change, 0 to destroy.
+
+──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
+
+Note: You didn't use the -out option to save this plan, so Terraform can't guarantee to take exactly these actions if you run "terraform apply" now.
+```
+
+# Terraform ready to apply
+
+Terraform executes the actions proposed in the Terraform plan and updates the resources. Run the command: **terraform apply** and type **yes** when asked to **Enter a value**.
```

Comment:
Here, you should definitely put "apply" right after "Terraform" at the beginning of the sentence so it reads:
Terraform apply executes the actions...

Take out the colon after "Run the command". Just put it in as a regular sentence:
Run the command **teraform apply** and type **yes** when asked to **Enter a value**.